presentation/matrix_view.py

```diff
@@ -91,10 +105,10 @@ def make_hover_text(mesh: MeshResults, matrix: Matrix) -> List[List[str]]:
         for col in row.columns:
 
             latency_ms = matrix.cells[row.agent_alias][col.agent_alias].latency_microsec.value * 1e-3
-            jitter = matrix.cells[row.agent_alias][col.agent_alias].jitter_microsec.value
+            jitter = matrix.cells[row.agent_alias][col.agent_alias].jitter_microsec.value * 1e-3
```

Comment:
```suggestion
            jitter_ms = matrix.cells[row.agent_alias][col.agent_alias].jitter_microsec.value * 1e-3
```
Because we already have latency_ms (which is good)

Reply:
Done